README.md

```diff
@@ -1,347 +1,514 @@
-* The subgroups city file contains multiple observations per census place per year. The file is long and the multiple observations per census place per year are for subgroups like race/ethnicity and poverty status.
+-   The subgroups county files contain multiple observations per county per year. These files are long and have multiple observations per county per year are for subgroups like race/ethnicity and poverty status.
+
+| year | state | county | state_name |   county_name    |  subgroup_type   |           subgroup            |
+|:--------:|:--------:|:--------:|:--------:|:--------:|:--------:|:-----------:|
+| 2014 |  01   |  001   | "Alabama"  | "Autauga County" |      "all"       |             "All"             |
+| 2014 |  01   |  001   | "Alabama"  | "Autauga County" | "race-ethnicity" |     "Black, Non-Hispanic"     |
+| 2014 |  01   |  001   | "Alabama"  | "Autauga County" | "race-ethnicity" |          "Hispanic"           |
+| 2014 |  01   |  001   | "Alabama"  | "Autauga County" | "race-ethnicity" | "Other Races and Ethnicities" |
+| 2014 |  01   |  001   | "Alabama"  | "Autauga County" | "race-ethnicity" |     "White, Non-Hispanic"     |
+
+-   The subgroups city files contain multiple observations per census place per year. These files are long and have multiple observations per census place per year are for subgroups like race/ethnicity and poverty status.
 
 # Project Organization
 
-* Each domain should have its own directory. The name of the directory should only contain lower case letters, numbers, and hyphens. Do not include spaces. 
-* The [overview documentation file](https://github.com/UI-Research/mobility-from-poverty/blob/main/documentation/overview.md) includes information about the metrics. It should contain clear instructions for running the code. It should contain a brief list of the assumptions and methodology used to create each metric.
-* Avoid absolute file paths. If using R, use `.Rproj`. If using Stata, use projects. Otherwise, set the working directory. This ensures that the code is portable. 
-* **Only add final data to the repository.** Each subfolder should contain a `data/` folder for intermediate data files. The `data/` folder should be added to the `.gitignore`. The final file should be added to GitHub. 
-* If possible, download your data with code or pull your data from an API with code. 
-* **Do not include any credentials in the repository.** Please reach out to [Aaron R. Williams](awilliams@urban.org) if this creates issues. 
-* Use names that play well with default ordering (e.g. 01, 02 and YYYY-MM-DD dates) for directory and file names.
+-   Each domain should have its own directory. The name of the directory should only contain lower case letters, numbers, and hyphens. Do not include spaces.
```

Comment:
Each domain should have its own directory (e.g. 01_financial-well-being, 02_housing, etc.).